docs/trace/extending-the-sdk/README.md

```diff
@@ -197,7 +197,7 @@ client .NET Core](../../../src/OpenTelemetry.Instrumentation.Http/README.md) .
 Instrumentation libraries for these are already provided in this repo. The
 [OpenTelemetry .NET
 Contrib](https://github.com/open-telemetry/opentelemetry-dotnet-contrib)
-repostory also has instrumentations for libraries like
+repository also has instrumentations for libraries like
```

Comment:
did you catch it manually or some tools flagged it? (wonder why the CI checks didn't catch it)

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/issues/3031